tests/envcheck.py

```
@@ -0,0 +1,21 @@
+import os
+
+def get_database_url():
+    """接続情報からDBのURLを生成する
+    """
+    MYSQL_DATABASE = os.environ.get("MYSQL_DATABASE")
```

Pylint (3.3.7)

[convention] Variable name "MYSQL_DATABASE" doesn't conform to snake_case naming style

(C0103)